QUEST.tsv

```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
-QUEST_20150714_002093	I've studied magics for a long time as a Sorcerer.{nl}From the process, the advancement of Sorcerer can't be explained without relating with the demons, and since the demons and the goddesses were facing against each other, the field has been widened to the studying of the goddesses.
-QUEST_20150714_002094	While I was doing that, I found out about the field of accumulation.{nl}That is an empty field. The theory of the start and the end is the fanstastical library which is at the boundary of transcendency for one purpose.
-QUEST_20150714_002095	The records about the disciple, Raius
-QUEST_20150714_002096	My another disciple Raius is clever and understands well, but his greed for knowledge is too much. I am worried that he may ruin the task.{nl}A scholar should always try to learn, but obsessing about it will make him walk the wrong path. {nl}
-QUEST_20150714_002097	But, Raius is the disciple I acknowledge.{nl}His passion towards me and knowledge is just being expressed in a wrong way.{nl}
-QUEST_20150714_002098	I hope Raius realizes himself.{nl}Since he is my discipe. he would be able to do it.
-QUEST_20150714_002099	The records of the disciple, Hones
-QUEST_20150714_002100	Among my disciples, two of them followed me until my latter years and they will be my successors after my death.
+QUEST_20150428_001951	$Born as a son of a guard. Completed the construction of Tenet Cathedral. He rests here.
+QUEST_20150428_001952	$Mother, please don't cry. {nl}I am finally free of the curse.
+QUEST_20150428_001953	$Finally, you sleep like you are at your mother's house. {nl}You don't have to worry about the pain any more. {nl}You don't have to wait for the salvation from the Goddess.
+QUEST_20150428_001954	$In the touch of the Goddess. Beside the mother. {nl}Sleeping. Having a sleep. {nl}Me, who is not beside, getting lean and hardened like a winter tree.
+QUEST_20150428_001955	$The sons of the guards rest here.
+QUEST_20150428_001956	$The Grass of Zachariel King
+QUEST_20150428_001957	$The king told those who engraved the letters and the symbols onto the hard woods {nl}to abstain from sin, but they lost their memories as the king ordered. {nl}I did it because I was afraid that the demons and the people with malicious intent may steal my artifacts. {nl}However, I don't want to repeat that for my people.
+QUEST_20150428_001958	$(The book has rotted so much that nothing inside can be read.)
+QUEST_20150428_001959	$The minister said, {nl}"We already heard the words of His Majesty. {nl}We don't know exactly why, but the reason this royal tomb was built was not to show off, {nl}but to prepare for the forecast that may bring chaos to the kingdom and change the future of the humans. {nl}If we don't do this important task, who will?
+QUEST_20150428_001960	$Warning Sign for Unauthorized Entrants
+QUEST_20150428_001961	$To protect the sacred secret of Zachariel King, no one may enter this facility.
```

Comment:
Missed that one yesterday: should change to "the sacred secret of King Zachariel"